cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizer.java

```diff
@@ -278,4 +331,66 @@ private String getRequestingUser() throws IllegalArgumentException {
     LOG.trace("Got requesting principal {}", principal);
     return principal.getName();
   }
+
+  @VisibleForTesting
+  public Map<AuthorizationPrivilege, Boolean> cacheAsMap() {
```

Comment:
This needs to be where ever the cache is because different extensions can have their own caching scheme. In this case, they happen to be the same but in general, cdap should not be dictating how the extensions should cache.